sourcegraph/about-replatform
# Fix MD posts

Closes #56 by updating all inline HTML in MD to work with our new MD/MDX parsing pipeline.

## To Test
- Run dev server, spot check that blogs load and there aren't console errors.
- Run static build and check that build runs successfully. Spot check records
